daemon/api_users.go

```diff
@@ -342,10 +356,10 @@ func createUser(c *Command, createData postUserCreateData) Response {
 		return InternalError("%s", err)
 	}
 
-	return SyncResponse(&userResponseData{
+	return SyncResponse([]userResponseData{{
```

Comment:
Is this an API break?

Reply:
I think, while desired, this is an API break and we should be more creative at to how it is introduced. Perhaps an extra header or some request option. I don't think we can just break this without consequences today.

Reply:
On deeper look, perhaps it is not? Wasn't createUser just introduced a moment ago?

Reply:
it isn't an API break (unless you count changes between revisions only on master to be such): the `/v2/create-user` endpoint preserves the existing (inconsistent) behaviour. This establishes the new `/v2/users` api to be "always returns a slice of userResponseData" which seems a lot saner to me.

Reply:
\o/ thank you!